### DISCUSSION TITLE
Allow review comments location to be beside the code and not over it

### DISCUSSION THREAD

**Discussion by zamir-ivry-silk** [community/community#42214] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Current situation is that review comments appear below the commented line. 
This makes it harder to see the code when reading/writing the comment. 
There is the possibility to disable 'show comments' but that hides the comments until one wants to comment and then they all appear again. 
I think a better approach is  to highlight the commented line but have the comment beside the code and not over it. 
that makes reviewing easier.

**Comment by Soham-Chakraborty-8455** [comment]
Okay
